langchain-core/src/utils/tiktoken.ts

```diff
@@ -1,44 +1,34 @@
 import {
   Tiktoken,
-  TiktokenBPE,
   TiktokenEncoding,
   TiktokenModel,
   getEncodingNameForModel,
 } from "js-tiktoken/lite";
 import { AsyncCaller } from "./async_caller.js";
 
-const cache: Record<string, Promise<TiktokenBPE>> = {};
+const cache: Record<string, Promise<Tiktoken>> = {};
 
 const caller = /* #__PURE__ */ new AsyncCaller({});
 
 export async function getEncoding(
   encoding: TiktokenEncoding,
-  options?: {
-    signal?: AbortSignal;
-    extendedSpecialTokens?: Record<string, number>;
```

Comment:
Were we never using these properties?

Reply:
Seems like we're only using for estimate token counts in `BaseLanguageModel::getNumTokens`, which does not handle AbortSignal as well.